vault/resource_jwt_auth_backend_test.go

```diff
@@ -72,6 +72,61 @@ func TestAccJWTAuthBackend_OIDC(t *testing.T) {
 	})
 }
 
+// TODO: OIDC plugin has a validation stage when configs are written,
+// and since we use circleci Docker, we can't mount files nor can it see a
+// local one. This will need to be updated when we switch to a Circle machine
+// executor.
+
+// The random numbers on the provider_config are a hash of the object.
+func TestAccJWTAuthBackend_OIDC_Provider_ConfigAzure(t *testing.T) {
+	path := acctest.RandomWithPrefix("oidc")
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testJWTLocal(t) },
```

Comment:
The azure provider_config doesn't do anything in the initialization, so it doesn't need to be skipped in CI. 

(The diff looks strange here, but I'm just suggesting to remove the todo comment block, and replace the PreCheck call to `testJWTLocal(t)` with `testAccPreCheck(t)`.)
 
```suggestion
// The random numbers on the provider_config are a hash of the object.
func TestAccJWTAuthBackend_OIDC_Provider_ConfigAzure(t *testing.T) {
	path := acctest.RandomWithPrefix("oidc")
	resource.Test(t, resource.TestCase{
		PreCheck:     func() { testAccPreCheck(t) },
```